devsetup/Makefile

```diff
@@ -199,13 +199,17 @@ edpm_deploy_instance: ## Spin a instance on edpm node
 	-oc cp scripts/edpm-deploy-instance.sh openstackclient:/tmp/
 	-oc rsh openstackclient bash /tmp/edpm-deploy-instance.sh
 
+.PHONY: standalone_deploy
+standalone_deploy: export COMPUTE_CEPH_ENABLED=${EDPM_COMPUTE_CEPH_ENABLED}
+standalone_deploy: export STANDALONE=true
+standalone_deploy: export INTERFACE_MTU=${NETWORK_MTU}
+	$(eval $(call vars))
```

Comment:
It worked for me if I add `standalone_deploy:` here. Makefile errors are really cryptic...

Reply:
yes it worked locally thanks for your time @cescgina++ 

```
.PHONY: standalone_deploy
standalone_deploy: export COMPUTE_CEPH_ENABLED=${EDPM_COMPUTE_CEPH_ENABLED}
standalone_deploy: export STANDALONE=true
standalone_deploy: export INTERFACE_MTU=${NETWORK_MTU}
standalone_deploy:
	$(eval $(call vars))
	scripts/standalone.sh ${EDPM_COMPUTE_SUFFIX} ${STANDALONE_COMPUTE_DRIVER} '${EDPM_COMPUTE_ADDITIONAL_NETWORKS}'

.PHONY: standalone
standalone: edpm_compute ## Create standalone VM
standalone: standalone_deploy
	$(eval $(call vars))
```